### PR TITLE
Feature/dql2 13 ckanext harvest setup

### PR DESCRIPTION
**Pull request by duttonw**
tweaks ontop of other work

### CODE REVIEW

File: recipes/ckanweb-deploy.rb [hunk 1]
```diff
@@ -322,3 +336,11 @@
 	group "root"
 	mode "0755"
 end
+
+# only have one server trigger harvest initiation, which then worker queues havester fetch/gather works through the queues.
+file "/etc/cron.hourly/ckan-harvest-run" do
+	content "/usr/local/bin/pick-job-server.sh && #{paster} --plugin=ckanext-harvest harvester run -c #{config_file} 2>&1 > /dev/null\n"
```
Comment (ThrawnCA) [inline recipes/ckanweb-deploy.rb:342]: I was under the impression that we don't need pick-job-server here.

File: files/default/supervisor-ckan-harvest-gather.conf [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+; ====================================================================
+; Supervisor configuration for CKAN Harvest background gather consumer
+; ====================================================================
+
+; 1. Copy this file to /etc/supervisor/conf.d
+; 2. Make sure the paths below match your setup
+
+
+[program:ckan-harvest-gather]
```
Comment (ThrawnCA) [inline files/default/supervisor-ckan-harvest-gather.conf:15]: Can we combine this with the "fetch" config file?

Comment (duttonw) [reply files/default/supervisor-ckan-harvest-gather.conf:15]: they are all independant queues and should have at least one per service. supervisord sock can handle all it seems.

File: recipes/ckanweb-deploy.rb (hunk 1 above)
Comment (duttonw) [reply recipes/ckanweb-deploy.rb:342]: the cron  job is to kick off once a day, it should have 3 try at the same time, this job adds to the queues which the supervisord processes go through. see https://confluence.csiro.au/display/seegrid/CKAN+Harvesting+User+Guide

**Pull request merged**